rollup.config.ts

```diff
@@ -0,0 +1,76 @@
+import nodeResolve from '@rollup/plugin-node-resolve'
```

Comment:
Checking: the `.spec.ts` files don't get picked up right?

Reply:
```
"exclude": ["src/**/*.spec.ts", "tests/**/*.*", "src/testing/asset.ts"]
```
We exclude them in `tsconfig.build.json`